README.md

```diff
@@ -139,23 +135,25 @@ that are using `wasi_snapshot_preview1` APIs.
 
 The `wasm-tools component new` subcommand takes an `--adapt` argument which acts
 as a way to polyfill non-component-model APIs, like `wasi_snapshot_preview1`,
-with component model APIs. The [preview2-prototyping] project is the current
-go-to location to acquire a polyfill from `wasi_snapshot_preview1` to an
-in-development version of "wasi preview2" which is specified with [WIT]
-and the component model.
-
-Notably you'll want to download [one of the adapter modules][preview1-build]
-and name it `wasi_snapshot_preview1.wasm` locally to pass as an `--adapt`
-argument to `wasm-tools component new`. Note that there are two modules
-provided on the downloads page, [one is for non-commands][non-command] which
-don't have a `_start` entrypoint in the generated core wasm module (e.g. the
-`cdylib` crate type in Rust) and [one that is for command modules][command]
-which has a `_start` entrypoint (e.g. a `src/main.rs` in Rust).
-
-[preview2-prototyping]: https://github.com/bytecodealliance/preview2-prototyping
+with component model APIs. The [Wasmtime] runtime publishes [adapter
+modules][preview1-build] with each release that are suitable to use with
+`--adapt` to implement `wasi_snapshot_preview1` in terms of WASI 0.2. On
+Wasmtime's releases page you'll see three modules to choose from:
+
+* [`wasi_snapshot_preview1.command.wasm`] - use this for CLI applications
+* [`wasi_snapshot_preview1.reactor.wasm`] - use this for applications that don't
+  have a `main` function for example
```

Comment:
```suggestion
  have a `main` function. For example, a process that responds to an event 
```

Not sure if that is 100% accurate, the situation I always thought of here was the http response, but that seems to have its only polyfill now but it would be nice to have a concrete example